vendor/bat-native-ledger/src/bat/ledger/internal/wallet/wallet.h

```diff
@@ -86,13 +86,11 @@ class Wallet {
 
   std::string GetClaimPayload(
       const std::string user_funds,
-      const std::string new_address,
+      const std::string address,
```

Comment:
All of these parameters should probably be const references.

Reply:
done